package.json

```diff
@@ -3,10 +3,12 @@
   "version": "0.7.0",
   "description": "Provider plugin for the Serverless Framework v1.x which adds support for Azure Functions.",
   "license": "MIT",
-  "main": "index.js",
+  "main": "./lib/index.js",
   "author": "Azure Functions",
   "scripts": {
-    "test": "eslint **/*.js"
+    "test": "eslint src/**/*.js",
+    "prebuild": "rm lib/ -rf && npm test",
```

Comment:
nit:`rm -rf lib/`

Reply:
Any advantage of putting flags before?  My habit is to put them after.